Cargo.lock

```diff
@@ -257,7 +257,7 @@ dependencies = [
  "criterion 0.2.11 (registry+https://github.com/rust-lang/crates.io-index)",
  "digest 0.8.1 (registry+https://github.com/rust-lang/crates.io-index)",
  "env_logger 0.6.2 (registry+https://github.com/rust-lang/crates.io-index)",
- "ethbloom 0.6.4 (registry+https://github.com/rust-lang/crates.io-index)",
+ "ethbloom 0.5.3 (registry+https://github.com/rust-lang/crates.io-index)",
```

Comment:
it seems this reverts the current master -- e.g. https://github.com/crypto-com/chain/pull/178

Reply:
i modified and merged with latest